modules/ob-release-notes-1.1.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-release-notes-1-1_{context}"]
+= Release notes for {builds-shortname} General Availability 1.1
+
+{builds-shortname} General Availability (GA) 1.1 is now available on {ocp-product-title} 4.16.
+
+[id="new-features-1-1_{context}"]
+== New features
+
+The following sections highlight what is new in {builds-shortname} 1.1.
+
+[id="builds-new-features-1-1_{context}"]
+=== Builds
+
+* The Shared Resource CSI Driver is now generally available, offering a stable solution for managing shared resources across Kubernetes clusters. With this release, the driver has permission to read all Kubernetes secrets.
+
+[id="new-features-1-1_{context}"]
+== Fixes
+
+* This release includes updates the `go-git package` to a later version. This update resolves the path traversal vulnerability by addressing the underlying issue in the ChrootOS implementation.
+
+* This release updates the `golang.org/x/net`` package to v0.23.0. This fix mitigates a potential DoS vulnerability in HTTP/2 handling by limiting the number of CONTINUATION frames processed.
```

Comment:
Maybe markup `CONTINUATION`? I see one other spot in our repo where someone wrote it as "`CONTINUATION` frames", FWIW.
```suggestion
* This release updates the `golang.org/x/net` package to v0.23.0. This fix mitigates a potential DoS vulnerability in HTTP/2 handling by limiting the number of `CONTINUATION` frames processed.
```